extensions/city-bikes/src/module.ts

```diff
@@ -0,0 +1,16 @@
+import { createExtension } from "@cognigy/extension-tools";
+
+import { getStationFromText } from "./nodes/getStationFromText";
+import { apiKeyConnection } from "./connections/apiKeyConnection";
+
+
+export default createExtension({
+	nodes: [
+
+		getStationFromText
+	],
+
+	connections: [
+		apiKeyConnection
+	]
+});
```

Comment:
Please add the following to the createExtension object:

```js
{
  options: {
    label: "City Bikes"
  }
}
```

Please see an example [here](https://github.com/Cognigy/Extensions/blob/master/extensions/service-now/src/module.ts#L55).

Reply:
Done